spec/controllers/v1.0/requests_controller_spec.rb

```diff
@@ -40,49 +48,111 @@
         expect(json['data'].size).to eq(5)
       end
     end
+
+    context 'when approver' do
+      let(:access_obj) { instance_double(RBAC::Access, :accessible? => true, :admin? => false, :approver? => true, :owner? => false) }
+      before do
+        allow(RBAC::Access).to receive(:new).with('requests', 'read').and_return(access_obj)
+        allow(access_obj).to receive(:process).and_return(access_obj)
+      end
+
+      it 'returns status code 403' do
+        get "#{api_version}/workflows/#{workflow_id}/requests", :headers => request_header
```

Comment:
@hsong-rh can the request_header be moved into a spec support file and shared across all tests. If we control it from a single file we can add new keys
https://github.com/ManageIQ/catalog-api/blob/cee85e285bec837b21cc0fe6723097bdfbd2539c/spec/support/service_spec_helper.rb#L7